terraform/fec_rds.tf

```diff
@@ -93,13 +93,21 @@ resource "aws_db_instance" "rds_production" {
   publicly_accessible = true
   storage_encrypted = true
   multi_az = true
+  auto_minor_version_upgrade = true
```

Comment:
Oh wow, this is not what I was seeing when I was looking at the docs before - somehow I was only seeing the parameter names, no values!